### PR TITLE
General changes in order to make the app work better

### PR DESCRIPTION
**Pull request by Borjarguez**

### CODE REVIEW

File: webroot/css/scss/layout/_content.scss [hunk 1]
```diff
@@ -408,6 +408,10 @@ strong {
                         .red-icon {
                             color: red;
                         }
+
+                        .blue-icon {
+                            color: #0096FF;
```
Comment (LeeScoresby) [inline webroot/css/scss/layout/_content.scss:413]: can we use vars?

File: webroot/css/scss/layout/_nav.scss [hunk 2]
```diff
@@ -297,14 +297,15 @@ nav {
             min-width: 0;
             word-wrap: break-word;
             background-color: white;
-            border-radius: 10px;
             margin-bottom: 1.5rem;
             box-shadow: 0 20px 27px 0 rgb(0 0 0 / 5%);
```
Comment (LeeScoresby) [inline webroot/css/scss/layout/_nav.scss:301]: can we encapsulate?

File: webroot/css/scss/layout/_nav.scss [hunk 3]
```diff
@@ -340,6 +342,8 @@ nav {
                 justify-content: center;
                 padding: 2rem;
```
Comment (LeeScoresby) [inline webroot/css/scss/layout/_nav.scss:343]: can we use vars for spacing and unify px/rem/em?

**Review by LeeScoresby — CHANGES_REQUESTED** [review]
Good in general but please, invest 30 min in copy-replace static values like colors and gaps into vars, 🙏

File: webroot/css/scss/layout/_content.scss (hunk 1 above)
Comment (Borjarguez) [reply webroot/css/scss/layout/_content.scss:413]: This has been improved yesterday. Changes will be added soon

File: webroot/css/scss/layout/_nav.scss (hunk 2 above)
Comment (Borjarguez) [reply webroot/css/scss/layout/_nav.scss:301]: Variables will be used to avoid using pixels

File: webroot/css/scss/layout/_nav.scss (hunk 3 above)
Comment (Borjarguez) [reply webroot/css/scss/layout/_nav.scss:343]: Of course!!

**Review by LeeScoresby — APPROVED** [review]

**Pull request merged**